test/server.js

```diff
@@ -1263,4 +1263,104 @@ describe('server block cache', function() {
     })
   })
 
-})
\ No newline at end of file
+})
+
+describe('server LRU', function() {
```

Comment:
I am not sure about the naming for these test cases. In [line 1152](https://github.com/mcollina/node-coap/pull/256/files#diff-d14cbfb50ed1cad7db5f4fef6a6076961b7cc9be980a3be06a70998f0eb8ebceR1152), there is already a `server LRU` present. Maybe the tests can be merged?

Reply:
Merging is one option, having the name state an "whats the test focus on" infgo is also an idea. tests should not become too complex imo

Reply:
> , having the name state an "whats the test focus on" infgo is also an idea. tests should not become too complex imo

Yeah, you are right. What do you think about this one?

```suggestion
describe('Client Identifier', function() {
```

 (Not sure why I haven't been thinking about this name earlier 😅)